wifi-subsys/test/dependencies.lock

```diff
@@ -0,0 +1,15 @@
+dependencies:
+  espressif/cbor:
+    component_hash: 46f10fad442a9edf81bfafff7550a41277363f710ad6a48f87316bb1016afed8
+    source:
+      service_url: https://api.components.espressif.com/
+      type: service
+    version: 0.5.4
+  idf:
+    component_hash: null
+    source:
+      type: idf
+    version: 5.0.0
+manifest_hash: cbf68eaa6a45e0d806821655af9f819fc8a95f52334076795faa4e4ee899c87a
+target: esp32
+version: 1.0.0
```

Comment:
This file is required? or to be ignored?. I am confuse with the file .gitignore that refers to this file..

Reply:
this should be ignored i think